### PR TITLE
adjusting consistency tests to remove column with a live date in it

### PR DESCRIPTION
**Pull request by wcornwell**

### CODE REVIEW

File: vignettes/ausflora.Rmd
Comment (fontikar) [inline vignettes/ausflora.Rmd:1]: @wcornwell instead could you set the chunk as `eval=FALSE`? 🙏

Comment (fontikar) [reply vignettes/ausflora.Rmd:1]: If its uncommented will it fail? Just my work on `website` branch will conflict a little but no biggy

Comment (wcornwell) [reply vignettes/ausflora.Rmd:1]: > load_taxonomic_resources(stable_or_current_data="stable") 
is fine 
it's just 
> load_taxonomic_resources(stable_or_current_data="current")
that needs their server up.

it's just that it seems like their downtime is like 20% of the time and so it'll create random failures for us if our testing and website building is dependent on their server being up.  And it's not really a core function for us anyway.

Comment (wcornwell) [reply vignettes/ausflora.Rmd:1]: i will switch to `eval=FALSE` and see if that also passes

Comment (wcornwell) [reply vignettes/ausflora.Rmd:1]: worked

**Review by fontikar — APPROVED** [review]
Wahoo!

**Comment by fontikar** [comment]
Once it all passes squash and merge 🚀 !!

**Pull request merged**